app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -167,4 +179,25 @@ class FeatureFlagNotificationViewModel @Inject constructor(
         }
         featureFlagState = featureFlagState.copy(shouldShowGuestRoomLinkDialog = false)
     }
+
+    fun getE2EICertificate() {
+        // TODO do the magic
```

Comment:
I suppose then for this PR it should be only about showing the dialogs, right?

Reply:
yeah, this one is only about informing user, and generating the Certificate will be implemented in other PR and I just put this `TODO` to make it easier to find where to call a usecase